docs/references/protocol/transactions/types/mptokenissuancecreate.md

```diff
@@ -60,6 +61,20 @@ Transactions of the MPTokenIssuanceCreate type support additional values in the
 | `tfMPTCanTransfer` | `0x00000020` | `32`          | If set, indicates that tokens can be transferred to other accounts that are not the issuer. |
 | `tfMPTCanClawback` | `0x00000040` | `64`          | If set, indicates that the issuer can use the `Clawback` transaction to claw back value from individual holders. |
 
+
+## Error Cases
+
+Besides errors that can occur for all transactions, {% $frontmatter.seo.title %} transactions can result in the following [transaction result codes](../transaction-results/index.md):
+
+| Error Code                | Description |
+|:--------------------------|:------------|
+| `tecDIR_FULL`             | The owner directory of the account creating the `MPTokenIssuance` ledger entry is full. |
+| `temBAD_TRANSFER_FEE`     | The transfer fee specified is greater than the maximum allowed value of 50,000. |
+| `temDISABLED`             | The `MPTokensV1` amendment is disabled. You will also receive this error if you include a `DomainID` field in the transaction, but the `PermissionedDomains` and `SingleAssetVault` amendments are both disabled. |
+| `tecINSUFFICIENT_RESERVE` | The account creating the `MPTokenIssuance` ledger entry doesn't have enough XRP to meet the owner reserve. |
+| `temMALFORMED`            | - A non-zero transfer fee is set, but the `tfMPTCanTransfer` flag is _not_ set.<br>- The `DomainID` points to an invalid permissioned domain; you can also receive this error if you include a `DomainID` without setting the `tfMPTRequireAuth` flag.<br>- The `MPTokenMetadata` field is an invalid length (0 or exceeds 1024 bytes).<br>- The `MaximumAmount` field is 0 or exceeds 9,223,372,036,854,775,807 (2^63-1). |
```

Comment:
`temMALFORMED` is one of the errors that can occur for any transaction type.

```suggestion
```

Reply:
True, but I think it's helpful to explain what specifically you could be malforming to make it easier for someone to troubleshoot.

Reply:
There are infinite ways to creatively malform a transaction, but if there are common uses or patterns that hit the specific cases described here, then I agree it makes sense to note it here with a preface like you suggested.